app/test_main.py

```diff
@@ -1,3 +1,13 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_proper_age_if_animal_years_are_more_than_14():
```

Comment:
why didn't you test all cases?

Reply:
I initially did, but since it was not a requirement of the test i decided not to include it.
I will redo i

Reply:
Done